extensions/cornerstone-dicom-sr/src/commandsModule.ts

```diff
@@ -123,6 +124,47 @@ const commandsModule = (props: withAppTypes) => {
         throw new Error(error.message || 'Error while saving the measurements.');
       }
     },
+
+    /**
+     * Loads measurements using either the given function or if the function is not present
+     * it hydrates and loads the SR for the given display set instance UID and displays
+     * it in the active viewport.
+     * @param loadMeasurementsEventFn the function to call to load the measurements
+     * @param loadMeasurementsEventName the name of the event to fire to load the measurements
+     * @param loadMeasurementsEventArgs arguments for the event; in particular the display set instance UID
+     * @param clearMeasurements optionally clear the measurements prior to hydrating the SR when loadMeasurementsEventFn is not present
+     * @returns
+     */
+    loadMeasurements: ({
+      loadMeasurementsEventFn,
+      loadMeasurementsEventName,
+      loadMeasurementsEventArgs,
+      clearMeasurements = true,
+    }) => {
+      if (loadMeasurementsEventFn) {
+        loadMeasurementsEventFn(loadMeasurementsEventName, loadMeasurementsEventArgs);
+        return;
+      }
```

Comment:
I don't understand this pattern. Why are we passing all this information just to call it immediately? Can't this be done outside in the `OHIFCornerstoneSRMeasurementViewport`?

Reply:
The idea was to move this code out of `OHIFCornerstoneSRMeasurementViewport` so that it can be more easily customizable in particular for those OHIF users that are NOT using tracked measurements. When not using tracked measurements, `loadMeasurementsEventFn` is undefined and the rest of the command is executed. Does this make sense?

Reply:
Even for those using tracked measurements, the ability to execute load or other measurements options tied to commands or other buttons means one can then avoid having to display the SR panel, and go directly to loading the SR data.  That means one could, for example, double click to load an SR series.

Reply:
It is just a strange pattern. I'm pretty sure there are better ways to do this, but I need to think about it. Maybe chop it into two things: one that is strange like this and the other that has the general logic regarding hydrateSR.

Reply:
The right way of doing it is to register a command of the same name in the tracked extension, and then just call it without context.  In a tracked setting, it would just call the right command automatically because of priority of calls.
In a non-tracked setting, it would call the baseline.   The command itself would directly invoke the useTracked updates to perform the load.

Reply:
@wayfarer3130 , I tried adding the same command (name) in both the `measurement-tracking` and the `cornerstone-dicom-sr` extensions. In the mode I was using BOTH extensions are loaded and used (i.e. I was simply using the `longitudinal` mode). However, I noticed that the command in `cornserstone-dicom-sr` was called and NOT the one in `measurement-tracking`. Could it be so because the `cornerstone-dicom-sr` extension is registered AFTER the `measurement-tracking` extension? Please advise.

Reply:
It is supposed to be called consistently in terms of ordering, in the order specified in the load order for the longitudinal mode.  Maybe that isn't happening correctly.  Hmm, the order is specified incorrectly in the load modules:
const extensionDependencies = {
  // Can derive the versions at least process.env.from npm_package_version
  '@ohif/extension-default': '^3.0.0',
  '@ohif/extension-cornerstone': '^3.0.0',
  '@ohif/extension-measurement-tracking': '^3.0.0',
  '@ohif/extension-cornerstone-dicom-sr': '^3.0.0',
  '@ohif/extension-cornerstone-dicom-seg': '^3.0.0',
  '@ohif/extension-cornerstone-dicom-pmap': '^3.0.0',
  '@ohif/extension-cornerstone-dicom-rt': '^3.0.0',
  '@ohif/extension-dicom-pdf': '^3.0.1',
  '@ohif/extension-dicom-video': '^3.0.1',
};


See how the sr is after the measurement tracking extension.  If you swap them it should fix things, and the tracking does depend on the dicom-sr module, so they should be loaded int he correct order.